CHANGELOG.md

```diff
@@ -2,6 +2,12 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)# 
 
 
+# [v7.2.0]
```

Comment:
mark it as unreleased.

For example
[v7.2.0](unreleased)

Reply:
done